frontend/src/components/Navbar.css

```diff
@@ -0,0 +1,119 @@
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    background: transparent;
+    font-size: 1.25rem;
```

Comment:
font is too much bigger, keep it to 1.1rem

Reply:
changed the `font-size` to 1.1 rem

![image](https://github.com/user-attachments/assets/247e57cd-5151-422a-8a12-29bfed288a53)